setup.py

```diff
@@ -304,11 +314,20 @@ def run(self):
 
 # Loosen constraints to ensure dev versions still work
 if 'dev' in __version__:
-    reqs_to_loosen = {'python-lsp-server[all]', 'qtconsole', 'spyder-kernels'}
+    reqs_to_loosen = {
+        'python-lsp-server[all]',
+        'qtconsole',
+        'qtconsole-base',
+        'spyder-kernels',
+    }
     install_requires = [req for req in install_requires
                         if req.split(">")[0] not in reqs_to_loosen]
+
     install_requires.append('python-lsp-server[all]>=1.12.0,<1.14.0')
-    install_requires.append('qtconsole>=5.5.1,<5.7.0')
+    base = ''
+    if os.getenv('SPYDER_QT_BINDING', 'pyqt5').lower() == 'none':
```

Comment:
```suggestion
    if os.getenv('SPYDER_QT_BINDING', 'pyqt5').lower() == 'conda-forge':
```

To go with my suggestion above.